src/main/java/com/minecolonies/coremod/blocks/BlockHutDeliveryman.java

```diff
@@ -15,6 +15,10 @@ protected BlockHutDeliveryman()
         super();
     }
 
+    public String getJobName()
+    {
+        return "Deliveryman";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS3400)